L3-L4-OOP/L3-Zad.1-MovieTheater/src/test/java/pl/movietheater/Siting.java

```diff
@@ -0,0 +1,44 @@
+package pl.movietheater;
+
+public class Siting {
+    private int sitNumber;
+    private int sitRow;
+    private String status;
```

Comment:
skoro status jest możliwy jedynie 'zajety/wolny' to wydaje się, że można tu zastosować bardziej adekwatny typ danych niż string. ;)

Reply:
+ nie generuj od razu niepotrzebnie wszystkich setterów. Nie ma w zadaniu napisane, że 'sitNumber/sitRow' moze sie zmienić (po stworzeniu obiektu).